recipes/HelloReact/web/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>Render routes</title>
```

Comment:
I think I've made this mistake elsewhere, too. Could you update this title to `HelloReact` or something?